console/src/shared/utils/deriveAction.js

```diff
@@ -170,7 +177,10 @@ const deriveAction = (
       name: v.variable.name.value,
     };
     const argTypeMetadata = getAstTypeMetadata(v.type);
-    if (!inbuiltTypes[argTypeMetadata.typename]) {
+    if (
+      !inbuiltTypes[argTypeMetadata.typename] &&
+      !isHasuraScalar(argTypeMetadata.type)
```

Comment:
```suggestion
      !isHasuraScalar(argTypeMetadata.typename)
```

Reply:
Also @beerose, the `isScalarType` function handles `undefined` and `null` as well. So you can directly use `isScalar(allHasuraTypes[name])` in the `isHasuraScalar` function.